### PR TITLE
Add example for enabling RHEL 8 repo

### PR DESCRIPTION
**Pull request by jwbernin**
RHEL 8 repos won't enable properly if an arch is specified. This change demonstrates how to enable the BaseOS release version 8 repository.

### CODE REVIEW

File: plugins/modules/katello_repository_set.py [hunk 1]
```diff
@@ -108,6 +108,16 @@
     state: disabled
     repositories:
       - basearch: x86_64
+
+- name: "Enable RHEL 8 BaseOS RPMs repositoryi with label"
```
Comment (mdellweg) [inline plugins/modules/katello_repository_set.py]:
```suggestion
- name: "Enable RHEL 8 BaseOS RPMs repository with label"
```

**Comment by mdellweg** [comment]
I am not an expert here. Maybe 'description' is the right place for something like this.
Thoughts? @sean797 @akofink

**Comment by sean797** [comment]
I think the example is good, perhaps there is something we can do on the Katello side as well. @jwbernin Would you mind creating an issue? https://projects.theforeman.org/projects/katello/issues/new

**Comment by jwbernin** [comment]
@sean797 I'm not sure this is an issue on the katello side. If it is, I don't see what it is well enough to open an issue with any sort of specifics, being mostly unfamiliar with the katello project itself.

**Pull request merged**